docs/containers.conf.5.md

```diff
@@ -422,6 +422,25 @@ Name of destination for accessing the Podman service. See SERVICE DESTINATION TA
 The cgroup management implementation used for the runtime. Supports `cgroupfs`
 and `systemd`.
 
+**compat_api_enforce_docker_hub**=true
+
+Enforce using docker.io for completing short names in Podman's compatibility
+REST API. Note that this will ignore unqualified-search-registries and
+short-name aliases defined in containers-registries.conf(5).
```

Comment:
Did not know this ignored short-names, not so crazy about this, but I guess this is what we got.

Reply:
That was a deliberate decision since we'd otherwise not be Docker compatible.